scripts/router.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+mongoAdminUser=$1
+mongoAdminPasswd=$2
+fqdn=$3
+
+install_mongo3() {
+
+	#install
+	wget https://repo.mongodb.org/yum/redhat/7/mongodb-org/3.6/x86_64/RPMS/mongodb-org-mongos-3.6.17-1.el7.x86_64.rpm
+	wget https://repo.mongodb.org/yum/redhat/7/mongodb-org/3.6/x86_64/RPMS/mongodb-org-shell-3.6.17-1.el7.x86_64.rpm
+	wget https://repo.mongodb.org/yum/redhat/7/mongodb-org/3.6/x86_64/RPMS/mongodb-org-server-3.6.17-1.el7.x86_64.rpm
+	rpm -i mongodb-org-mongos-3.6.17-1.el7.x86_64.rpm
+	rpm -i mongodb-org-shell-3.6.17-1.el7.x86_64.rpm
+	rpm -i mongodb-org-server-3.6.17-1.el7.x86_64.rpm
+	PATH=$PATH:/usr/bin; export PATH
+
+	#disable selinux
+	sed -i 's/^SELINUX=.*/SELINUX=disabled/' /etc/sysconfig/selinux
+	sed -i 's/^SELINUX=.*/SELINUX=disabled/' /etc/selinux/config
+	setenforce 0
+
+	#kernel settings
+	if [[ -f /sys/kernel/mm/transparent_hugepage/enabled ]];then
+		echo never > /sys/kernel/mm/transparent_hugepage/enabled
+	fi
+	if [[ -f /sys/kernel/mm/transparent_hugepage/defrag ]];then
+		echo never > /sys/kernel/mm/transparent_hugepage/defrag
+	fi
+
+	#configure
+	sed -i 's/\(bindIp\)/#\1/' /etc/mongod.conf
+
+	#set keyfile
+	echo "vfr4CDE1" > /etc/mongokeyfile
```

Comment:
how is this value "vfr4CDE1" used?

Reply:
this is used in the Mongokeyfile that is used by the nodes to trust and talk to one another